tests/cupyx_tests/scipy_tests/linalg_tests/test_special_matrices.py

```diff
@@ -62,6 +63,8 @@ class TestSpecialMatrices(TestSpecialMatricesBase):
     @testing.numpy_cupy_allclose(atol=1e-5, rtol=1e-5, scipy_name='scp',
                                  accept_error=ValueError)
     def test_special_matrix(self, xp, scp):
+        if self.function == "kron":
+            warnings.filterwarnings('ignore', category=DeprecationWarning)
```

Comment:
Please avoid modifying warning filters globally as it affects following tests.

Reply:
Fixed. b997443